accepted/future-releases/control-flow-collections/implementation-plan.md

```diff
@@ -149,6 +155,11 @@ this to handle the new syntax.
 
 [atom]: https://github.com/dart-atom/dart
 
+### VS Code
```

Comment:
cc @dantup (work hasn't started here yet; when it does, we may need to update the grammar file)

Reply:
I pasted the samples from this file into VS Code and it seems like this may be handled as we'd want without any changes:

<img width="475" alt="screenshot 2019-01-08 at 10 46 15 am" src="https://user-images.githubusercontent.com/1078012/50826140-c079a180-1332-11e9-848a-6737ebf481ec.png">